File: python/src/tests/t_get_env.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

atrshmlog.flush()

maxindex = atrshmlog.get_statistics_max_index()

stats = atrshmlog.get_statistics()

print('flush : ' + str(stats[80]) + ' : ')

print (' ')

exit(0)

# end of test

File: perl/src/tests/t_get_env.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$s = shift @ARGV;

$val = Atrshmlog::get_env($s);

print "get env : $val : \n";

$val = Atrshmlog::get_env_shmid();

print "get env : $val : \n";

$val = Atrshmlog::get_env_id_suffix();

print "get env : $val : \n";


print " \n";

exit (0);

# end of main
